pkg/controller/cheinstallation/cheinstallation_controller.go

```diff
@@ -106,40 +118,47 @@ func (r *ReconcileCheInstallation) Reconcile(request reconcile.Request) (reconci
 		return reconcile.Result{}, err
 	}
 
-	err = r.ensureCheInstallation(reqLogger, cheInstallation)
+	requeue, err := r.ensureCheInstallation(reqLogger, cheInstallation)
+	if requeue {
+		return reconcile.Result{Requeue: true, RequeueAfter: time.Second * 5}, err
```

Comment:
I thought that we don't want to return the error, but only log it and reque with error=nil

Reply:
yeah, but that was meant only for the case when the CheCluster is not found yet and still waiting for being provisioned, but if the watch call fails, then we need to know that it failed

Reply:
Yes, return error will log the error and thus can be analysed.